go/libkb/id_table.go

```diff
@@ -141,6 +141,7 @@ type RemoteProofChainLink interface {
 	ComputeTrackDiff(tl *TrackLookup) TrackDiff
 	GetProofType() keybase1.ProofType
 	ProofText() string
+	RequiresHint() bool
```

Comment:
not sure if this is the right move since we should be getting hints back from the server once the hunter is in..